acct-group/gamemode/gamemode-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for the system-wide games-util/gamemode resource group"
+ACCT_GROUP_ID=501
```

Comment:
Switched to 501 according to https://wiki.gentoo.org/wiki/Proxied_Maintainer_FAQ#Adding_acct-.7Bgroup.2Cuser.7D.2Fpackages_as_a_proxied_maintainer